sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -848,3 +848,15 @@ CREATE TABLE `benefit_eligibility` (
 #IfTable eligibility_response
 DROP TABLE `eligibility_response`;
 #Endif
+
+#IfTable x12_partners
+ALTER TABLE `x12_partners` CHANGE `processing_format` `processing_format` ENUM('standard','medi-cal','cms','proxymed','oa-eligibility','avality-eligibility') DEFAULT NULL;
+#Endif
+
+#IfMissingColumn eligibility_id insurance_companies
+ALTER TABLE `insurance_companies` ADD `eligibility_id` VARCHAR(32) DEFAULT NULL;
+#Endif
+
+#IfMissingColumn x12_default_eligibility_id insurance_companies
+ALTER TABLE `insurance_companies` ADD `x12_default_eligibility_id` INT(11) NULL AFTER `eligibility_id`;
```

Comment:
I always avoid the AFTER stuff since you just never know (of the column it is placed after somehow does not exist; even though in this case it is literally impossible to not exist because it is added above) and it doesn't impact anything if it happens to not be placed AFTER a certain element.

Reply:
good point